### ISSUE TITLE
Sufficient Description of Light Spectrum Box?

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#245]
For https://github.com/phetsims/QA/issues/461. Found in MacOS VO.
Currently, when examining the light spectrum dialog box, you hear:
>The show light spectrum button opens a window displaying the EM spectrum with the values of the frequency and wavelength shown. Additional arrows show the direction of increasing frequency/energy and wavelength. A sketch of a wave with decreasing wavelength/increasing frequency is also provided.

Knowing how frequency, energy, and wavelength relate to the parts of the spectrum may very well be necessary for students. As such, I don't think this should be so vague. At the very, very least it should say something about microwave having low energy/high wavelength and Ultraviolet having high energy/low wavelength.

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe - Indeed, what is in the sim is pretty old.

I noticed there is more content for this in the design doc @BLFiedler @Matthew-phet @terracoda can you comment on whether or not this is complete/ready to go into the sim?

**Comment by terracoda** [comment]
I can’t comment, @Matthew-phet or @BLFiedler can comment on readiness of Light Spectrum description? 

If not ready, it can wait until next round of interviews.

**Comment by BLFiedler** [comment]
I think @Matthew-phet was going to pretty it up, but let me know if you need some help!

**Comment by BLFiedler** [comment]
Okay - I fixed up the ranges under H2. @terracoda, can you check that a screen reader will read out the ranges correctly? Not sure what the best nomenclature is for exponents. 

Unassigning for now, feel free to reassign if you need help @Matthew-phet

**Comment by jessegreenberg** [comment]
Discussed during meeting, for tomorrow I will add this content:
H1: Light Spectrum
P: The Light Spectrum shows the relative energy of the different classifications of light waves as defined by their characteristic wavelengths (measured in meters) and frequencies (measured in Hertz or inverse seconds). The order from lowest energy (lowest frequency and largest wavelength) to highest energy (highest frequency and smallest wavelength) is Radio, Microwave, Infrared, Visible, Ultraviolet, X-ray, and Gamma ray.

A sine wave [waveform] decreasing in wavelength (as measured by the distance from peak to peak) and increasing frequency (as measured by the number of waves per time interval) from Radio to Gamma Ray.

The following content involves scientific notation and we need more time for that.

**Comment by jessegreenberg** [comment]
https://github.com/phetsims/molecules-and-light/issues/245#issuecomment-566697059 content was added to sim.

We will continue with the scientific notation portions outside of sim.

**Comment by terracoda** [comment]
BTW, our consultant liked the general description of light spectrum a lot. We just need to figure how to present the valued range of. She said it might be useful to describe the ranges in terms of an approximate magnitude (e.g. the rainbow of visible light occupies a tiny part of the spectrum between blah and blah).

**Comment by BLFiedler** [comment]
Do you think she meant in a way that adds additional information to the ranges or would replacing the existing ranges with more approximate values (and qualitative description) be better? This ties to my thought that the exact ranges are not very meaningful without some quick mental math and that stating order of magnitude ranges might be better.

**Comment by terracoda** [comment]
I think she meant that a qualitative description of the ranges would be more useful, in general. 

That said, I think if we have numeric ranges in the visual, we have to provide both in the non-visual. The visual has both. The visual has a spacial and numeric representations.
